Zend/zend_closures.c

```diff
@@ -147,7 +147,7 @@ ZEND_METHOD(Closure, call)
 
 	fci.retval = &closure_result;
 	fci.params = my_params;
-	fci.param_count = my_param_count;
+	fci.param_count = -my_param_count;
```

Comment:
Did you eliminate copy in comparison to php-7? (I didn't understand this)

On Wed, Dec 16, 2015 at 5:38 PM, Bob Weinand <notifications@github.com>
wrote:

> In Zend/zend_closures.c
> <https://github.com/php/php-src/pull/1679#discussion_r47782276>:
>
> > @@ -147,7 +147,7 @@ ZEND_METHOD(Closure, call)
> >
> >  	fci.retval = &closure_result;
> >  	fci.params = my_params;
> > -	fci.param_count = my_param_count;
> > +	fci.param_count = -my_param_count;
>
> Small note: negative param count is a hack to avoid copying around. I'm
> open for other suggestions. (let's discuss that later.)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/php/php-src/pull/1679/files#r47782276>.
>